### PR TITLE
feat: add `ctx context.Context` snippet

### PR DESCRIPTION
**Pull request by fpuc**
Hi @honnamkuan. I have added a small snippet for completing `ctx context.Context`

### CODE REVIEW

File: snippets/snippets.code-snippets [hunk 1]
```diff
@@ -81,6 +81,13 @@
     "body": ["fmt.Errorf(\"${1:%+v}\", ${0:err})"]
   },
 
+  "ctx context.Context": {
+		"prefix": "ctx",
```
Comment (hnkuanx) [inline snippets/snippets.code-snippets]: Can you indent with 2 spaces instead of tabs?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Below is a list of all available snippets and the triggers of each one.
 | `cl`   | close `close(closable)` |
 | `fms`   | fmt Sprintf `fmt.Sprintf("%+v", args)` |
 | `fme`   | fmt Errorf `fmt.Errorf("%+v", args)` |
+| `ctx`   | ctx context.Context` |
```
Comment (hnkuanx) [inline README.md]: Please remove the trailing backtick `

Make it
```
| `ctx` | ctx context.Context |
```

**Review by hnkuanx — CHANGES_REQUESTED** [review]
Left feedback as inline comments

File: snippets/snippets.code-snippets (hunk 1 above)
Comment (fpuc) [reply snippets/snippets.code-snippets]: :+1: updated

File: README.md (hunk 2 above)
Comment (fpuc) [reply README.md]: also updated TY

File: snippets/snippets.code-snippets [hunk 3]
```diff
@@ -81,6 +81,13 @@
     "body": ["fmt.Errorf(\"${1:%+v}\", ${0:err})"]
   },
 
+  "ctx context.Context": {
+    "prefix": "ctx",
+    "body": [
+      "ctx context.Context",
+    ],
+	},
```
Comment (hnkuanx) [inline snippets/snippets.code-snippets]: Can you replace the tabs with spaces here as well?

**Review by hnkuanx — CHANGES_REQUESTED** [review]

**Review by hnkuanx — APPROVED** [review]

**Pull request merged**